src/bandersnatch/package.py

```diff
@@ -64,28 +66,18 @@ def normalized_simple_directory(self) -> Path:
             )
         return self.mirror.webdir / "simple" / self.normalized_name
```

Comment:
so the difference here is the legacy format used a parent directory with the first letter as the top level directory name? I just want to be sure I understand correctly what is going away

Reply:
As we discussed on IRC - Just the PEP's normalization function rather than some arbitrary one from back in the day in distutils.

This will also cause less to be on disk.